superset/config.py

```diff
@@ -1371,6 +1371,21 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Set to False to only allow viewing own recent activity
 ENABLE_BROAD_ACTIVITY_ACCESS = True
 
+# Configuration for environment tag shown on the navbar. Setting 'text' to '' will hide the tag.
+ENVIRONMENT_TAG_CONFIG = {
+    "variable": "FLASK_ENV",
+    "values": {
+        "development": {
+            "color": "#c73d2e",
+            "text": "Development",
+        },
+        "production": {
+            "color": "#039dfc",
+            "text": "Production",
+        },
+    },
+}
+
```

Comment:
Okay, so in this case, we would have to override the `ENVIRONMENT_TAG_CONFIG` in the `hogwarts-superset-deployment` repo and outline two/three new variables:

`PB`
`U`

MAYBE: `Development` (which would be applied to the local dev container)

We wouldn't need `UDev`, or `PB-stg` since they would be at the U or PB classification respectively. 

Again, this would need changes to the `hogwarts-superset-deployment` repo and the dev container repo, especially if we are going to use a new env variable instead of `FLASK_ENV`.